source/getting-started/quickstart.md

```diff
@@ -0,0 +1,262 @@
+This guide will teach you how to build a simple app using Ember from
+scratch.
+
+We'll cover these steps:
+
+1. Installing Ember
+2. Creating a new application
+3. Defining a route
+4. Writing a UI component
+5. Building your app to be deployed to production
+
+## Install Ember
+
+You can install Ember with a single command using npm, the Node.js package
+manager. Type this into your terminal:
+
+```sh
+npm i ember-cli -g
+```
+
+Don't have npm? [Learn how to install Node.js and npm here][npm].
+
+[npm]: https://docs.npmjs.com/getting-started/installing-node
+
+## Create a New Application
+
+Once you've installed Ember via npm, you will have access to a new
+`ember` command in your terminal. You can use the `ember new` command to
+create a new application.
+
+```sh
+ember new ember-quickstart
+```
+
+This one command will create a new directory called `ember-quickstart`
+and set up a new Ember application inside of it.  Out of the box, your
+application will include:
+
+* A development server
+* Template compilation
+* CSS preprocessing
+* JavaScript and CSS minification
+* ES2015 features via Babel
+
+By providing everything you need to build production-ready web
+applications in an integrated package, Ember makes starting new projects
+a breeze.
+
+Let's make sure everything is working properly. Open the
+`ember-quickstart` directory in your favorite editor, such as Sublime
+Text or Vim. Once you have your editor open, `cd` into the application
+directory and start the development server by typing:
+
+```sh
+cd ember-quickstart
+ember serve
+```
+
+After a few seconds, you should see output that looks like this:
+
+```text
+Livereload server on http://localhost:49152
+Serving on http://localhost:4200/
+```
+
+(To stop the server at any time, type Ctrl-C in your terminal.)
```

Comment:
@trek is correct, but @maclover7 does raise the good point that we don't tell newbs how to kill the server once they've started it. :stuck_out_tongue: Maybe we should open an issue or PR on ember-cli.